collectors/hra/parser.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from .record import Record
+from datetime import date
+import logging
+import six
+logger = logging.getLogger(__name__)
+
+
+def parse_response(response, endpoint, from_date, to_date, errors, success, conn):
```

Comment:
Ideally we don't need args like errors, success, conn here. Also could `endpoint` be extracted from `response`?